### ISSUE TITLE
Feature request: Disabling denoise

### ISSUE THREAD

**Issue by hrj** [labthings/picamerax#8]
According to the discussion [here](https://github.com/raspberrypi/userland/issues/490), it is possible to suppress denoising by setting parameter `MMAL_PARAMETER_STILLS_DENOISE` to `MMAL_FALSE`.

Can you please expose this through `picamerax`? It will help immensely with low-light photography.

While we are at it, being able to set `MMAL_PARAMETER_CAMERA_ISP_BLOCK_OVERRIDE` would be also great. Setting it to `~0x80` apparently disables pixel-correction.

thanks!

**Comment by jtc42** [comment]
Yeah I’m so sorry I only just saw this. I somehow disabled notifications for this repo and so didn’t see the issue.

I have really limited time to spend on this now, but I’ll certainly look into it. @rwb27 may be able to help too, but again it may be way out of scope.

The other option may be that IIRC the raw bayer data doesn’t include denoising (I may be wrong though) so if that’s the case you can use the bayer data to get what you need.

**Comment by rwb27** [comment]
Yes, the Bayer data is pre-denoising.  If it's a simple parameter to add, it shouldn't be a big job; it's worth noting, though, that the Pi foundation have now got a `libcamera` interface, which feels like it solves most of the problems we are trying to fix with `picamerax` - so it might be in the near future that we can abandon `picamerax` in favour of that other, much clearer interface.

I'll leave this open and look at it if I can.
